### PR TITLE
web ui test admin general settings system status

### PR DESCRIPTION
**Pull request by paurakhsharma**
## Description
This PR adds tests for system status in Admin General Settings Page

Related issue: #32851

## How Has This Been Tested?
Locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Acceptance test

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: tests/acceptance/features/bootstrap/WebUIAdminGeneralSettingsContext.php [hunk 1]
```diff
@@ -198,6 +198,28 @@ public function before(BeforeScenarioScope $scope) {
 		}
 	}
 
+	/**
+	 * @Then the version of owncloud installation should be displayed on the admin general settings page
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/WebUIAdminGeneralSettingsContext.php]: the version of the owncloud installation

File: tests/acceptance/features/lib/AdminGeneralSettingsPage.php [hunk 2]
```diff
@@ -201,23 +204,71 @@ public function setUpdateChannelValue($updateChannel) {
...
+	 * @return string ownCloud Version String
+	 */
+	public function getOwncloudVersionString() {
+		$actualVersion = $this->find("xpath", $this->ownCloudVersionStringXpath);
+		if ($actualVersion === null) {
 			throw new ElementNotFoundException(
 				__METHOD__ .
-				" xpath $selectCron->getXpath() " .
-				"could not find xpath for radio button"
+				" xpath $this->ownCloudVersionXpath " .
```
Comment (phil-davis) [inline tests/acceptance/features/lib/AdminGeneralSettingsPage.php]: $this->ownCloudVersionStringXpath

**Review by phil-davis — CHANGES_REQUESTED** [review]
See review comments

**Comment by paurakhsharma** [comment]
Backport on: #33005

**Review by phil-davis — APPROVED** [review]

**Pull request merged**